dell/src/main/java/org/apache/iceberg/dell/EcsClientProperties.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.dell;
+
+/**
+ * Property constants of catalog
+ */
+public interface EcsClientProperties {
+
+  /**
+   * Access key id
+   */
+  String ACCESS_KEY_ID = "ecs.s3.access.key.id";
+
+  /**
+   * Secret access key
+   */
+  String SECRET_ACCESS_KEY = "ecs.s3.secret.access.key";
+
+  /**
+   * S3 endpoint
+   */
+  String ENDPOINT = "ecs.s3.endpoint";
+
+  /**
+   * Factory class of {@link EcsClientFactory}.
+   * <p>
+   * The config is optional. If properties above aren't enough, use this.
```

Comment:
when people read this in generated javadoc web page, they cannot see what are the "properties above". Please see the documentation in `AwsProperties` to improve the javadoc.